### PR TITLE
new version, with new file sebastian2.txt

### PR DESCRIPTION
**Pull request by k-sebastian**
I want to merge this verison with master

### CODE REVIEW

**Comment by k-sebastian** [comment]
I want really merge this pleeeeeeazzzz

File: sebastian2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+new file, for the branch
```
Comment (k-markus) [inline sebastian2.txt]: that looks awesome!!! Good Work!

Comment (k-markus) [inline sebastian2.txt]: Better now!

**Comment by k-markus** [comment]
Ok, lets do it

**Pull request merged**